pyproject.toml

```
@@ -119,6 +119,8 @@ filterwarnings = [
   # elevate conda's deprecated warning to an error
   "error::PendingDeprecationWarning:conda",
   "error::DeprecationWarning:conda",
+  # someone inserted a space...
+  "ignore::PendingDeprecationWarning: conda",
```

We should be more specific with the ignore here and move this down to the other conda ignore on line 139:

```suggestion
  "ignore:conda.trust is pending deprecation:PendingDeprecationWarning:conda",
```